server/index.js

```diff
@@ -1,18 +1,37 @@
 const express = require('express');
 // var bodyParser = require('body-parser');
+const db = require('../database/index.js');
 let app = express();
 
+
 // app.use(bodyParser.json());
-// app.use(bodyParser.urlencoded({extended: false}))
+app.use(express.urlencoded({extended: false}))
 
 app.use(express.static(__dirname + '/../client/dist'));
 
-app.post('/', function (req, res) {
+app.get('/images/mainImages/:productID', (req, res) => {
+
+  db.read(req.params.productID, 'mainImages')
+    .then(images=> {
+      res.send(images)
+    })
+    .catch(err =>console.log(err))
+})
 
+app.get('/images/thumbnailImages/:productID', function (req, res) {
+  db.read(req.params.productID, 'thumbnailImages')
+  .then(images=> {
+    res.send(images)
+  })
+  .catch(err =>console.log(err))
 })
 
-app.get('/', function (req, res) {
-// res.send('yay')
+app.get('/images/detailImages/:productID', function (req, res) {
+  db.read(req.params.productID, 'detailImages')
+  .then(images=> {
+    res.send(images)
+  })
+  .catch(err =>console.log(err))
```

Comment:
lol. i like your res.send('yay') that you commented out! 
Here, you may want to also send like status code 404 or something along that line. So if there is an error, the user is not just sitting there and wondering and you have a console.log(). 
Also, i think you can add your node_modules in your .gitignore file. So you don't push them.
Overall, great job!!!